core/services/gateway/connectionmanager.go

```diff
@@ -52,6 +53,18 @@ type connectionManager struct {
 	lggr               logger.Logger
 }
 
+func (m *connectionManager) HealthReport() map[string]error {
```

Comment:
Can we add coverage to the new API surface?

Reply:
Maybe. I'd rather not repeat something trivial at a low level all over though.

It would be nice to have a high level test that spins up many things and provides a view of all the different kinds of services from one place.
There is a `/health` endpoint that we can test against.

Another approach is to eliminate the need to write this method every time. I just revived an old service helper branch that might help with this, by providing an extension of `utils.StartStopOnce` that incorporates this, among other things.

Reply:
https://github.com/smartcontractkit/chainlink/pull/10697

Reply:
Ah, makes sense. I like that approach